### PR TITLE
update(JS): web/javascript/reference/global_objects/string/tolocaleuppercase

### PR DESCRIPTION
**Pull request by undead404**
Оригінальний вміст: [String.prototype.toLocaleUpperCase()@MDN](https://developer.mozilla.org/en-us/docs/Web/JavaScript/Reference/Global_Objects/String/toLocaleUpperCase), [сирці String.prototype.toLocaleUpperCase()@GitHub](https://github.com/mdn/content/blob/main/files/en-us/web/javascript/reference/global_objects/string/tolocaleuppercase/index.md)

Нові зміни:
- [mdn/content@fb85334](https://github.com/mdn/content/commit/fb85334ffa4a2c88d209b1074909bee0e0abd57a)
- [mdn/content@c2445ce](https://github.com/mdn/content/commit/c2445ce1dc3a0170e2fbfdbee10e18a7455c2282)
- [mdn/content@76198d4](https://github.com/mdn/content/commit/76198d446ea5e46d5321a409f89c3c64573b86e1)
- [mdn/content@b7ca46c](https://github.com/mdn/content/commit/b7ca46c94631967ecd9ce0fe36579be334a01275)
- [mdn/content@b5c766f](https://github.com/mdn/content/commit/b5c766f4eecb4fcf9d8ba175caddb94f7c3e9d20)
- [mdn/content@5e20796](https://github.com/mdn/content/commit/5e207965797b3672d3c06b65298de551d1eac515)
- [mdn/content@a92a2bb](https://github.com/mdn/content/commit/a92a2bb31cf5d79808878701f0344a4eabf12963)
- [mdn/content@9be220f](https://github.com/mdn/content/commit/9be220f47da9a5f8cb36dc05ba49b19b9929dffa)
- [mdn/content@f3df525](https://github.com/mdn/content/commit/f3df52530f974e26dd3b14f9e8d42061826dea20)
- [mdn/content@2eb202a](https://github.com/mdn/content/commit/2eb202adbe3d83292500ed46344d63fbbae410b5)
- [mdn/content@8db2031](https://github.com/mdn/content/commit/8db2031e46de297b9a02dca59eb8d8767fe6c61e)
- [mdn/content@ce29091](https://github.com/mdn/content/commit/ce2909126eb09e44c9f48d9f65d072acae827749)
- [mdn/content@968e6f1](https://github.com/mdn/content/commit/968e6f1f3b6f977a09e116a0ac552459b741eac3)
- [mdn/content@71576a9](https://github.com/mdn/content/commit/71576a92c3c835c1ffea4fd2150beb7dbb1e2553)
- [mdn/content@d6f63f6](https://github.com/mdn/content/commit/d6f63f6e41146f459e21046af1c51fadb772de70)
- [mdn/content@a6785a8](https://github.com/mdn/content/commit/a6785a8e4ddad80069acf2de418235c0640688e7)
- [mdn/content@4c9f9bb](https://github.com/mdn/content/commit/4c9f9bb287b37743a0e626fd2b60f54240f4f07e)
- [mdn/content@edaa4af](https://github.com/mdn/content/commit/edaa4aff4cf7532efdcba6014b9316aad2a5b6f1)
- [mdn/content@281e3b2](https://github.com/mdn/content/commit/281e3b21178946c8301232a8eb50d11770ee8450)

### CODE REVIEW

**Review by AdriandeCita — CHANGES_REQUESTED** [review]
виглядає ок, але є свіжі зміни

File: files/uk/web/javascript/reference/global_objects/string/tolocaleuppercase/index.md [hunk 1]
```diff
@@ -1,46 +1,37 @@
...
-  - Reference
-  - String
+page-type: javascript-instance-method
 browser-compat: javascript.builtins.String.toLocaleUpperCase
 ---
+
 {{JSRef}}
 
-Метод **`toLocaleUpperCase()`** повертає значення рядка, на якому викликається, переведене у верхній регістр із врахуванням будь-яких, специфічних для конкретної локалі, перетворень.
+Метод **`toLocaleUpperCase()`** (до вищого регістру згідно з локаллю) значень {{jsxref("String")}} повертає свій рядок, переведений у вищий регістр, з врахуванням будь-яких перетворень, специфічних для конкретної локалі.
```
Comment (KlashReik) [inline files/uk/web/javascript/reference/global_objects/string/tolocaleuppercase/index.md:10]:
```suggestion
Метод **`toLocaleUpperCase()`** (до вищого регістру згідно з локаллю) значень {{jsxref("String")}} повертає свій рядок, переведений у вищий регістр, із врахуванням будь-яких перетворень, специфічних для конкретної локалі.
```

File: files/uk/web/javascript/reference/global_objects/string/tolocaleuppercase/index.md [hunk 2]
```diff
@@ -1,46 +1,37 @@
...
-toLocaleUpperCase([locale1, locale2, ...])
+toLocaleUpperCase(locales)
 ```
 
 ### Параметри
 
-- `locale` {{optional_inline}}
-  - : Параметр `locale` позначає локаль, налаштування якої будуть використані для переведення рядка у верхній регістр, з урахуванням будь-яких, специфічних для локалі, перетворень. В разі, якщо було передано декілька локалей у {{jsxref("Array", "масиві")}}, буде використано [найкращу наявну локаль (англ.)](https://tc39.github.io/ecma402/#sec-bestavailablelocale). Усталена локаль — це локаль середовища хост-системи.
+- `locales` {{optional_inline}}
+  - : Рядок з тегом мови BCP 47, або масив таких рядків. Позначає локаль, налаштування якої будуть використані для переведення рядка у верхній регістр, з урахуванням будь-яких, специфічних для локалі, перетворень. Загальну форму та тлумачення аргументу `locales` дивіться в [описі цього параметра на головній сторінці `Intl`](/uk/docs/Web/JavaScript/Reference/Global_Objects/Intl#arhument-locales).
```
Comment (KlashReik) [inline files/uk/web/javascript/reference/global_objects/string/tolocaleuppercase/index.md:24]:
```suggestion
  - : Рядок з тегом мови BCP 47, або масив таких рядків. Позначає локаль, налаштування якої будуть використані для переведення рядка у верхній регістр, з урахуванням будь-яких специфічних для локалі перетворень. Загальну форму та тлумачення аргументу `locales` дивіться в [описі цього параметра на головній сторінці `Intl`](/uk/docs/Web/JavaScript/Reference/Global_Objects/Intl#arhument-locales).
```

**Review by KlashReik — CHANGES_REQUESTED** [review]
пара зауважень

**Review by KlashReik — APPROVED** [review]

**Pull request merged**